packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/ReferencesButton.css

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright (c) 2019 Uber Technologies, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.multi-parent-button {
+  padding: 0 5px;
+  color: #000;
+}
+
+.external-trace-ref {
+  float: right;
```

Comment:
`float` should be avoided. Once I submit I'll link how I recommended getting around `float` in another file in this PR.